phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/application/skin/ApplicationInformationPanelSkin.java

```diff
@@ -247,4 +273,14 @@ private void installScript(ScriptDTO script) {
                     }
                 }));
     }
+
+    private void assignIcon(Label label, String tooltip, String imageName, double height) {
+        String imageDir = "/org/phoenicis/javafx/filter/";
+        Image image = new Image(getClass().getResourceAsStream(imageDir + imageName));
+        ImageView iv = new ImageView(image);
+        iv.setFitHeight(height);
+        iv.setFitWidth(height);
+        label.setGraphic(iv);
+        label.setTooltip(new Tooltip(tr(tooltip)));
+    }
```

Comment:
Our normal approach is to set images using css. This gives us two benefits:

1. it allows us to change an image independently of the "main" application code
2. it allows us to define multiple independent look and feels that use different images for the same button/element.

It would be nice if you could change your code so that it uses css to define the images :)

An example how we did this for the library buttons can be found here:

https://github.com/PhoenicisOrg/phoenicis/blob/c313db12ec1ace9cf60cb2ccebba86a39cfd7d12/phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/standard/main.less#L900-L902

and here:

https://github.com/PhoenicisOrg/phoenicis/blob/ed8d91e34379915a531006d7f504bb93d2976bb4/phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/skin/ShortcutInformationPanelSkin.java#L161-L165

It should be possible to do the same (or something similar) with other components than `Button`.